cfg/phab.json

```diff
@@ -133,7 +133,7 @@
 
   // pre-subfilter triangulation options
   "triangulation": {
-    "method": 1,
+    "method": "linf_angular",
```

Comment:
Please also make the same change to `cfg/tumvi_cam0.json` and `cfg/tumvi_cam1.json`.